### DISCUSSION TITLE
Time to drop annotations

### DISCUSSION THREAD

**Discussion by fogrye** [thecodingmachine/graphqlite#673] [General]
Doctrine annotations was deprecated since php 8.0 release, should we drop support of them as php7 is not supported and there are no other reason not to use attributes?

**Comment by oojacoboo** [comment]
Yes, they've been deprecated for quite a bit now.  I think now that 7.0 has been released, now is a suitable time to replace them with PHP attributes and remove the Doctrine annotation lib as a dependency.
